src/react/databrowser/buckets/details/Workflow.tsx

```diff
@@ -0,0 +1,120 @@
+import { useHistory } from 'react-router';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import Table from '@scality/core-ui/dist/components/tablev2/Tablev2.component';
+import { Button } from '@scality/core-ui/dist/next';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+import Icon from '@scality/core-ui/dist/components/icon/Icon.component';
+import { TextTransformer } from '../../../ui-elements/Utility';
+import { useWorkflows } from '../../../workflow/Workflows';
+import { AppState } from '../../../../types/state';
+import { makeWorkflows } from '../../../queries';
+import { APIWorkflows } from '../../../../types/workflow';
+import { NameLinkContaner } from '../../../ui-elements/NameLink';
+import { push } from 'connected-react-router';
+
+const TableAction = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: ${spacing.sp16};
+`;
+
+function Workflow({ bucketName }: { bucketName: string }) {
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const accountName = useSelector(
+    (state: AppState) =>
+      state.auth.selectedAccount && state.auth.selectedAccount.userName,
+  );
+  const select = (workflows: APIWorkflows) => makeWorkflows(workflows);
+  const { data, status } = useWorkflows(select, [bucketName]);
+
+  const nameCell = (value) => {
+    const id = value.row.original.id;
+    const workflowName = value.value;
+    return (
+      <NameLinkContaner
+        onClick={() =>
+          dispatch(push(`/accounts/${accountName}/workflows/${id}`))
```

Comment:
we don't need to dispatch here. We can simply do 
```suggestion
          history.push(`/accounts/${accountName}/workflows/${id}`)
```